### DISCUSSION TITLE
Some doubts about the LAGRANGE  method  to estimate the pose from points.

### DISCUSSION THREAD

**Discussion by 867415148** [lagadic/visp#1291] [Q&A]
Hello, I recently found the code for the Lagrangian method for solving the position from a point (From  poseLagrangeNonPlan()) when I was learning about position estimation algorithms. I am unable to understand the formulas represented by these codes, can you please recommend me some papers to help me understand the Lagrangian method codes or briefly explain the principles of the codes. Thank you very much and have a nice life.

**Comment by 867415148** [comment]
@fspindle Hello Professor, I would be very grateful if you could answer this question in your spare time, thank you!

**Comment by fspindle** [comment]
See [#1295](https://github.com/lagadic/visp/issues/1295)
